docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/skip-link/skip-link.twig

```diff
@@ -17,8 +17,11 @@
 
 {% if text is not empty %}
   <div class="civictheme-skip-link {{ modifier_class }}">
-    <a class="civictheme-skip-link__link civictheme-visually-hidden focusable" href="{{ url }}">
-      {{ text }}
-    </a>
+    {% include '@atoms/content-link/content-link.twig' with {
+      theme: theme,
+      text: text,
+      url: url,
+      modifier_class: 'civictheme-skip-link__link civictheme-visually-hidden focusable',
```

Comment:
could you please update `focusable` to `civictheme-focusable` here and in all components where it is used

Reply:
Updated focusable to civictheme-focusable.
Only a single reference was found.